modules/client-configuration/pages/contact-methods-saltssh.adoc

```diff
@@ -1,73 +1,111 @@
 [[contact-methods-saltssh]]
-= Push via Salt SSH
+= SSH Push contact method
 
 
-Push via Salt SSH is used in environments where Salt clients cannot reach the {productname} Server directly.
+SSH Push ([literal]``ssh-push``) is used in environments where Salt clients cannot reach the {productname} Server directly.
 In this environment, clients are located in a firewall-protected zone called a DMZ.
 No system within the DMZ is authorized to open a connection to the internal network  where the {productname} Server is located.
 
-Push via Salt SSH is also to use if installing a daemon agent on clients is not possible.
+SSH Push is also to use if installing a daemon agent on clients is not possible.
 
 [WARNING]
 ====
-The Push via Salt SSH method has serious limitations.
-It does not scale well, and consumes more Server resources and network bandwidth than the plain Salt method.
+The SSH Push method has serious limitations.
+It does not scale well, and consumes more Server resources and network bandwidth than the plain Salt method ([literal]``default``).
 The Push via Salt SSH method is not at all supported with large setups (1000 clients and more).
 ====
 
-The Push via Salt SSH method creates an encrypted tunnel from the {productname} Server on the internal network to the clients located in the DMZ.
-After all actions such as updates and events are pushed and executed, the tunnel is closed.
+The server uses the SSH Push to contact the clients at regular intervals, checking in and performing scheduled actions and events.
 
-The server uses the Salt SSH to contact the clients at regular intervals, checking in and performing scheduled actions and events.
-
-////
-This contact method works for Salt clients only.
-For traditional clients, use Push via SSH.
-////
-
-This image demonstrates the Push via Salt SSH process path.
+This image demonstrates the Push via SSH Push process path.
 All items left of the [systemitem]``Taskomatic`` block represent processes running on the {productname} client.
 
 image::salt-ssh-contact-taigon.png[scaledwidth=80%]
 
 
-To use Push via Salt SSH, you must have the SSH daemon running on the client, and reachable by the [systemitem]``salt-api`` daemon running on the {productname} Server.
-Additionally, the required Python version will be installed with the salt-bundle on the remote system.
+To use SSH Push, you must have the SSH daemon running on the client, and reachable by the [systemitem]``salt-api`` daemon running on the {productname} Server.
+Additionally, the required Python version will be installed with the {salt} Bundle on the client system.
```

Comment:
If we are talking about salt-ssh systems here it's not fully correct to use `installed`, as the python is not installed there as a package, but maybe use `deployed` as we are deploying the salt codebase and all the requirements including python to `/var/tmp/...` only, so it's temporary and can be redeployed on next session.

Reply:
ok